scripts/bwctest_osd.sh

```diff
@@ -97,65 +101,54 @@ done
 [ -z "$BIND_PORT" ] && BIND_PORT="5601"
 [ -z "$VERSIONS" ] && test_array=("${DEFAULT_VERSIONS[@]}") || IFS=',' read -r -a test_array <<<"$VERSIONS"
```

Comment:
Can `test_array` be SCREAMING_SNAKE_CASE like the other global variables? While we're at it, maybe rename it to something like `TEST_VERSIONS` to remove ambiguity?

Reply:
Good catch! Yeah I prefer that actually, makes a lot more sense.